### DISCUSSION TITLE
Can't get Sidekick to work.

### DISCUSSION THREAD

**Discussion by daver99999** [frntc/Sidekick64#90] [Q&A]
I built one of these and when I load it. I get the TFT display but the computer doesn't boot right. Comes up with a blue screen but no cursor or basic message. Any Ideas?

**Comment by SensoryTrash** [comment]
check it for bad solder joints and especially the RPI connector are you getting led Cyclin ok?

upload pictures so we can see if anything obvious shows.

**Reply by daver99999** [reply]
I checked the soldering and none of the pins move at all. I don't see any shorts. Here are some pictures. My SMD soldering is not great. 
![20230916_120822](https://github.com/frntc/Sidekick64/assets/29669101/a322a546-0d5c-4e7f-b344-8630273cd78e)
![20230916_120809](https://github.com/frntc/Sidekick64/assets/29669101/391453ac-71a1-4177-a9d8-63d78b531d3b)
![20230916_120813](https://github.com/frntc/Sidekick64/assets/29669101/73ebeb34-478c-4d79-b724-baaf8a1b6c2f)

**Reply by daver99999** [reply]
Also the lights blink as normal.

**Reply by daver99999** [reply]
I built a new one and it works fine now. Don't know what I did wrong.

**Comment by SensoryTrash** [comment]
573a looks slightly miss aligned but should be ok...

whats the value for SMD Resistors on below Leds ive got 1081

so screen is good and shows logo leds cycle... what happens when you press reset?  black screen? or normal 

what happens when you press menu button?>

you are powering the pi as well externally ?

**Reply by daver99999** [reply]
The resisters are 182's 
Yes screen is good and LED's cycle.
Pressing Reset seems to reset the C64 IE Lt and right borders get narrower then back to Original screen with no cursor or basic message.
Pressing the menu button does nothing.
Yes the PI is powered externally.

**Reply by SensoryTrash** [reply]
well i dont think its your build as ive built 3 of them and all have the identical problem and im not hand soldering im reflowing the, in a oven...

where did you get the pcbs pressed ????

**Reply by daver99999** [reply]
PCBWAY

**Reply by SensoryTrash** [reply]
so it seems we both have the same problem. im wondering if its the pcb?

**Reply by SensoryTrash** [reply]
or crappy chips

**Reply by daver99999** [reply]
Well That sucks did you get your boards from PCBWAY?

**Comment by SensoryTrash** [comment]
and i changed my chips as well for ti and nexperia ones and the result is the same.

**Comment by SensoryTrash** [comment]
you did uncomment the correct line in the config,txt on the sd card?

just checking

**Reply by daver99999** [reply]
Yes.

**Comment by MC6809E089** [comment]
Probably a daft question but have you set the jumper for A13? I can't see it in your photos.

**Reply by daver99999** [reply]
Tried it with and without. Not supposed to be needed on a C64.

**Comment by pigravity** [comment]
I had somewhat similar situation 1/3 Sidekicks wasnt working, for no particular reason. Once ive resoldered all chips - as a test - to another PCB, everything was suddenly ok.. But im not ordering on PCBWAY.. So cant blame that :)

**Reply by SensoryTrash** [reply]
i have assembled 5 now and they all are the same,,,,,,

no menu.... everything else is fine...

**Comment by frntc** [comment]
Hi guys,

a quick begin of a checklist for trouble shooting:
- does the HDMI output show the logo? No => RPi doesn't boot, likely SD-card content is not correct
- does the TFT display show the logo after boot (but HDMI does) => LVC573
- are GAME and EXROM low upon start (measure at expansion port)? No => check LCX/LVC07 and make sure GPIO0 and 15 on the RPi are LOW)

According to my experience (and from the building thread on forum64.de) it usually is some solder bridge or missing contact with the SMD ICs...

**Reply by daver99999** [reply]
HDMI Logo Yes
TFT Shows Logo
GPIO0 and 15 are both Low
Game and EXROM are low till the C64 is turned on then they both go to about 2.3 Volts. Is that correct?

**Reply by SensoryTrash** [reply]
ok don't want to throw you guys off but i had a somewhat fix... i replaced LCX07G with the Nexperia 74LVC07AD and you can see what i used all NXP chips... not that it should make a difference.

i also re soldered all the pins on all ics just to be sure as i used a reflow oven...

![Sidekick Working](https://github.com/frntc/Sidekick64/assets/95701281/8f4c91c7-83df-4db9-9bab-0f4eb5b1aaa4)

i have re soldered the other boards using the LCX07G and there is no change non working...

**Reply by SensoryTrash** [reply]
Replaced  LCX07G  on non working board with NXP 74LVC07AD and its working so all things point to the LCX07G either a fake chip or not compatible...

omg im rocking out now to Hubbard Tunes.

**Reply by daver99999** [reply]
Not sure I understand.  You Replaced the LVX07G with a 245? How could that work? My lcx07G's came from Mouser so they shouldn't be fake.

**Reply by SensoryTrash** [reply]
Sorry for the confusion i replaced the LCX07G with the  74LVC07AD and it worked...

I also tested a board i built with the TI LVC145A and it also worked after replacing the LCX07G

**Reply by frntc** [reply]
I've build boards with LCX07 and they worked, but testing of GAME/EXROM is exactly what reveals if the bus signals are pulled down (required for starting the menu).

GAME/EXROM should remain low after boot... @daver99999 do you have a chance to test with a LVC07? If those are causing problems for some of you, I'd update the BOM on the github-page (and/or add a warning).

**Reply by daver99999** [reply]
Well I just tried replacing with the same chip and still not working for me.

**Reply by frntc** [reply]
replacing the LCX07 with a LVC07? ... or another LCX07?

**Reply by daver99999** [reply]
With a LVC07

**Reply by daver99999** [reply]
Let me check the Game and EXROM again with the LVC

**Reply by daver99999** [reply]
Game and Exrom are showing the same thing. 2.3ish volts.

**Reply by frntc** [reply]
... but they go low after reset at first, correct?

**Reply by daver99999** [reply]
Seems like they do.

**Comment by nathanozlegend** [comment]
I had the same issue found out my raspberry pi3/B was stuffed. I replaced it and it worked. but with your picture you are missing a jumper on A13

**Comment by SensoryTrash** [comment]
I tested with other Pis and J13 is optional but thanks :)

**Comment by lac463** [comment]
Might not help in this case, but the faults are similar.
So I had the blank blue screen. The 2nd and 4th LEDs on the sidekick were constantly on (sometimes the 1st as well). 
I have a C64C short board and a Pi 3A+. Originally I powered the Pi from my pc via USB, And I powered the C64 from a brick of death through tebl's power saver circuit.
I ordered my PCB from Restore Store, SMD assembled. So I ruled out SMD soldering problems and I checked my THT soldering multiple times with a multimeter.
I adjusted the config files.
Both the HDMI and the TFT screen had the SIdeKick logo showing, but I tried to use newer bootcode files anyway. Ofc it didn't work.
I checked Vcc and it was fine. I checked  GAME and EXROM on the expansion port, both were low.
I remembered, my cartridge port might be crusty from years of sitting, so I tried to wiggle an old cart in and out a few times but no dice.
I tried powering the Sidekick through the C64, while doing that, there was a point i didn't hook up the power sawer circuit and suddenly ths Sidekick started working. I was able to reproduce the error with the power saver after that.
It also worked while powering the Pi from my phone charger and powering the C64 with the power saver.
Also when is moved my C64, the blue screen came back. So yea, My cart port was also flakey.

TL;DR: Try different power sources for the Pi and contact cleaner on the cart port.

**Comment by sasquatchuk** [comment]
Hi I have the same problem did you get this to work. About to solder another one

**Reply by daver99999** [reply]
I never did get it to work. Was going to try another but I ended up in the hospital. Might try another this week.

**Reply by sasquatchuk** [reply]
Hi .
Thanks for the reply.. Hope your feeling better, I did actually build another and it worked fine apart from the tuneful 8 :). Here is my video https://youtu.be/UadJsf1-peQ

**Reply by daver99999** [reply]
I built a new one and it works fine now. Don't know what I did wrong.
